Remotion/Web/Development.WebTesting.IntegrationTests/WebTestHelperTest.cs

```diff
@@ -31,6 +32,38 @@ namespace Remotion.Web.Development.WebTesting.IntegrationTests
   [TestFixture]
   public class WebTestHelperTest
   {
+    private static readonly HashSet<string> s_relevantProcessNames = new()
+                                                                     {
+                                                                         "chromedriver",
+                                                                         "chrome",
+                                                                         "msedgedriver",
+                                                                         "edge",
+                                                                         "geckodriver",
+                                                                         "firefox"
+                                                                     };
+
+    // Filter for the driver process names that might be left open if something went wrong
+    private static readonly Func<Process, bool> s_processFilter = static p =>
+    {
+      return s_relevantProcessNames.Contains(p.ProcessName);
+    };
```

Comment:
Feels weird to me. Wouldn't a static method be easier and have the same effect?
```suggestion
    private static bool FilterBrowserProcesses (Process process)
    {
      return s_relevantProcessNames.Contains(process.ProcessName);
    };
```

And add the comment as xmlcomment instead of code comment.